index.html

```diff
@@ -61,27 +61,323 @@
+                </div>
+                <div class="card-icons">
+                  <i class="bi bi-heart"></i>
+                  <i class="bi bi-cart"></i>
+                </div>
+              </div>
+            </div>
+          </section>
+          <section class="component">
+            <div class="card__container">
+              <div class="card-header">
+                <img
+                  src="/images/pedigree-image.png"
+                  alt=""
+                  class="card-image"
+                />
+                <span class="card-badge">20% OFF</span>
+              </div>
+              <div class="card-details">
+                <h2 class="detail-heading">Pedigree for Dogs</h2>
+                <p class="detail-subheading">Adult 1.2 Kg</p>
+              </div>
+              <div class="card-footer">
+                <div class="price-segment">
+                  <span class="item-price">&#x20b9; 250</span>
+                  <span class="item-price dashed">&#x20b9; 350</span>
+                </div>
+                <div class="card-icons">
+                  <i class="bi bi-heart"></i>
+                  <i class="bi bi-cart"></i>
+                </div>
+              </div>
+            </div>
+          </section>
         </div>
       </main>
+
+      <footer class="footer">
+        <div class="footer-container">
+          <div class="social-media">
+            <div class="logo">
+              <a href="#"
+                ><img
+                  src="/images/PetmateLogo.png"
+                  alt="Petmate-logo"
+                  class="header__logo"
+              /></a>
+            </div>
+            <div class="footer-info">
+              <h3 class="footer-h3">One place shop to all pet lovers</h3>
+              <h3 class="footer-h3">@2022, All rights reserved</h3>
+              <h3 class="footer-h3">Made with love by Arjun</h3>
+            </div>
+            <div class="media-links">
+              <a href=""><i class="fab fa-twitter"></i></a>
+              <a href=""><i class="fab fa-linkedin"></i></a>
+              <a href=""><i class="fab fa-github"></i></a>
+            </div>
+          </div>
+          <div class="categories list">
+            <h2 class="list-h2">Categories</h2>
+            <ul>
+              <li>Cat Food</li>
+              <li>Dog Food</li>
+              <li>Treats</li>
+              <li>Toys</li>
+            </ul>
+          </div>
+          <div class="accounts list">
+            <h2 class="list-h2">Accounts</h2>
+            <ul>
+              <li>My Account</li>
+              <li>Wishlist</li>
+              <li>Basket</li>
+              <li>Privacy Policy</li>
+              <li>Refund Policy</li>
+            </ul>
+          </div>
+          <div class="contacts list">
+            <h2 class="list-h2">For Contact</h2>
+            <ul>
+              <li>9999911111</li>
+              <li>arjun9852@gmail.com</li>
```

Comment:
u can wrap it in a tag providing a tag so that if user click will open dialer and mail (href="tel:9888712344" and href="mailto:abc@gmail.com") just like this

Reply:
this was really cool. Thanks a lot.